### DISCUSSION TITLE
401 Unauthorized on helm install

### DISCUSSION THREAD

**Discussion by jvieten** [strimzi/strimzi-kafka-operator#9634] [General]
on 
`helm install strimzi-cluster-operator oci://quay.io/strimzi-helm/strimzi-kafka-operator`

I am getting the error : 

Error: unexpected status from HEAD request to https://quay.io/v2/strimzi-helm/strimzi-kafka-operator/strimzi-kafka-operator/manifests/0.39.0: 401 Unauthorized

Do I need an Red Hat account (paid) to pull the image?

**Comment by scholzj** [comment]
You should not need any account. It seems to work fine for me:

```
helm install strimzi-cluster-operator oci://quay.io/strimzi-helm/strimzi-kafka-operator
NAME: strimzi-cluster-operator
LAST DEPLOYED: Fri Feb  2 16:57:07 2024
NAMESPACE: myproject
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing strimzi-kafka-operator-0.39.0

To create a Kafka cluster refer to the following documentation.

https://strimzi.io/docs/operators/latest/deploying.html#deploying-cluster-operator-helm-chart-str
```

Maybe just some temporary glitch somwhere?

**Reply by jvieten** [reply]
Thanks for answering so quickly!  In my terraform  tf file  I had a typo which led to the "401 Unauthorized" . So I  made the wrong assumption and thought I needed authentication! Thanks alot!!
